custom-namespaces.yaml

```diff
@@ -213,6 +217,10 @@ contextual_services:
           table: mozdata.org_mozilla_ios_fennec.topsites_impression
       type: ping_view
   explores:
+    final_adM_forecasts:
+      type: ping_explore
+      views:
+        base_view: final_adM_forecasts
```

Comment:
```suggestion
```
If the spreadsheet is a `table_view` then lookml-generator currently does not generate explores for those. But you can create a custom one once the view has been deployed